doc/src/vpr/graphics.rst

```diff
@@ -56,16 +54,28 @@ Visualizing Placement
 --------------------------------
 By default VPR's graphics displays the FPGA floorplan (block grid) and current placement.
 
+.. figure:: ../Images/Overall_view.png
+    :align: center 
+    :height: 300
+
+    FPGA floorplan (block grid)    
+
+If the **Placement Macros** drop down is set, any placement macros (e.g. carry chains, which require specific relative placements between some blocks) will be highlighted.
+
 .. figure:: https://www.verilogtorouting.org/img/neuron_placement_macros.gif
     :align: center
 
     Placement with macros (carry chains) highlighted
 
-If the **Placement Macros** drop down is set, any placement macros (e.g. carry chains, which require specific relative placements between some blocks) will be highlighted.
-
 Visualizing Netlist Connectivity
 --------------------------------
-The **Toggle Nets** drop-down list toggles the nets in the circuit visible/invisible.
+The **Toggle Nets** drop-down list under the **Net Settings** tab toggles the nets in the circuit to be visible/invisible. Options include **Nets** and **Logical Connections**.
```

Comment:
Look in the draw code to figure out what logical connections are and describe them here.  Send snippets of code to us by email if needed; can use git blame to see who added the logical connection lines in the menu.

Reply:
Logical nets should be renamed --> suggest Primitive nets
Nets should be renamed --> suggest Cluster nets
Help should explain that showing the primitive nets showing connections within the cluster, and the primitive within the cluster that drives/receives them.  Cluster nets shows only the nets that go between clusters.